frontend/containers/impact-chart/component.tsx

```diff
@@ -128,11 +145,41 @@ export const ImpactChart: FC<ImpactChartProps> = ({
 
   if (compactMode) {
     return (
-      <div className={className}>
+      <div
+        className={classNames({
+          'w-full overflow-visible': !isPlaceholder,
+        })}
+        {...hoverProps}
+      >
         {isPlaceholder ? (
-          <div className="w-full border border-dashed rounded-full aspect-square bg-background-middle border-background-dark bg-radial-beige" />
+          <div className="w-full border border-dashed rounded-full aspect-square border-background-dark bg-radial-beige" />
         ) : (
-          <Radar data={chartData} options={chartOptions} />
+          <div>
+            <Tooltip
+              placement="top"
+              visible={isHovered}
+              content={
+                <div className="w-0 h-0 overflow-visible text-xs text-center text-gray-600 bg-red-700">
+                  <div className="px-1 -translate-y-3 -translate-x-[50%] bg-white border rounded-md w-min border-beige">
+                    {impactData[ImpactsEnum.Biodiversity].name}
+                  </div>
+                  <div className="px-1 -translate-x-[calc(100%+42px)] translate-y-5 bg-white border rounded-md w-min border-beige">
+                    {impactData[ImpactsEnum.Water].name}
+                  </div>
+                  <div className="px-1 bg-white border rounded-md translate-x-[42px] w-min border-beige">
+                    {impactData[ImpactsEnum.Climate].name}
+                  </div>
+                  <div className="px-1 translate-x-[-50%] translate-y-8 bg-white border rounded-md w-min border-beige">
+                    {impactData[ImpactsEnum.Community].name}
+                  </div>
+                </div>
```

Comment:
A few comments on the tooltips:
- The border radius is smaller, you should use `rounded` instead of `rounded-md`
- The padding is bigger: `px-2 py-1.5`
- The font size is smaller: `text-2xs`
- The colour is a lighter gray: `text-gray-400`
- The font is bolder: `font-semibold`
- You should remove the red background that you used for testing

Reply:
I also think you should create separate tooltips. This will allow you to have arrows on each of them.

For example, the top tooltip can be:
```jsx
            <Tooltip
              placement="top"
              arrow
              arrowClassName="border border-transparent border-b-beige border-r-beige"
              content={
                <div className="font-semibold text-center text-gray-400 text-2xs px-2 py-1.5 bg-white border rounded w-min border-beige">
                  {impactData[ImpactsEnum.Biodiversity].name}
                </div>
              }
            />
```

Inside of this, you would put the left tooltip, inside the right, etc. You will have to update the arrow slightly for each of them. For example, for the left tooltip, the `arrowClassName` prop should have this value: `-rotate-45 border border-transparent border-b-beige border-r-beige`.

Initially, this will not work because you need to make a small change to the `<Tooltip />` component so it passes a ref. Here is the details of the changes you need to make:
```jsx
import {  forwardRef } from 'react';

// More code

// Notice I've renamed this component
export const Inner: React.ForwardRefRenderFunction<any, TooltipProps> = (
  { children, content, arrow, maxWidth, arrowClassName, ...props },
  ref
) => {
  // More code

  return (
    <Tippy
      {...props}
      // Very important, set the `ref` prop on the `<div />`
      render={(attrs) => (
        <motion.div style={{ scale, opacity, maxWidth: maxWidth || 'none' }} {...attrs}>
          <div className="relative" ref={ref}>
            {/* More code */}
          </div>
        </motion.div>
      )}
      // More props here
    >
      {children}
    </Tippy>
  );
};

// You forward the ref here
export const Tooltip = forwardRef(Inner);

export default Tooltip;
```

Then don't forget to remove `useHover` from the the chart component.

Reply:
I couldn’t make it work this way, so I used the 'reference'